conda/recipes/libraft/meta.yaml

```
@@ -50,20 +50,25 @@ outputs:
         - ninja
         - sysroot_{{ target_platform }} {{ sysroot_version }}
       host:
-        - cuda-profiler-api {{ cuda_profiler_api_host_version }}
-        - cudatoolkit ={{ cuda_version }}
-        - libcublas {{ libcublas_host_version }}
-        - libcublas-dev {{ libcublas_host_version }}
-        - libcurand {{ libcurand_host_version }}
-        - libcurand-dev {{ libcurand_host_version }}
-        - libcusolver {{ libcusolver_host_version }}
-        - libcusolver-dev {{ libcusolver_host_version }}
-        - libcusparse {{ libcusparse_host_version }}
-        - libcusparse-dev {{ libcusparse_host_version }}
         - librmm ={{ minor_version }}
+        - cudatoolkit {{ cuda_spec }}
```

We want to tightly constrain host requirements to ensure that we build with CUDA 11.8 (the `cuda_version` used at build time across all of RAPIDS). We use `cuda_spec` for run dependencies where any compatible 11.x version will suffice.
```suggestion
        - cudatoolkit {{ cuda_version }}
```